### PR TITLE
Mkobos stylistical fixes

### PR DESCRIPTION
**Pull request by mkobos**
Simple pull request. Lots of changes but all of them are simple code reformattings, most of them done automatically.

### CODE REVIEW

File: vipe/graphviz/converter.py [hunk 1]
```diff
@@ -34,37 +35,37 @@ def __init__(self, detail_level, show_input_ports, show_output_ports,
...
+
         Args:
             pipeline (Pipeline): input Pipeline.
-        
+
         Return:
             string: dot format
         """
-        assert self.__already_run == False
+        assert self.__already_run is False
```
Comment (marekhorst) [inline vipe/graphviz/converter.py]: Isn't it possible to simply assert
`assert self.__already_run`
without explicitly comparing to boolean value? Using `not` when comparing to false.

**Comment by marekhorst** [comment]
TL;DR LGMT ;)

File: vipe/graphviz/converter.py (hunk 1 above)
Comment (mkobos) [inline vipe/graphviz/converter.py]: True. Missed that.

**Comment by mkobos** [comment]
> TL;DR LGMT ;)

LOL
